MTGOSDK/lib/ILRepack/ILRepack.targets

```diff
@@ -21,6 +23,7 @@
       OutputFile="$(OutputPath)$(AssemblyName).dll"
       InputAssemblies="@(InputAssemblies)"
       NoRepackRes="true"
+      DebugInfo="true"
```

Comment:
It seems that the ILRepack bunding of reference assemblies is not deterministic when enabling `DebugInfo`. Without this option though, SourceLink cannot find the package symbols.